lib/matplotlib/backend_tools.py

```diff
@@ -1020,6 +1021,75 @@ def _mouse_move(self, event):
         self.toolmanager.canvas.draw_idle()
 
 
+class HelpTool(ToolToggleBase):
```

Comment:
Should be ToolHelp for consistency (or rather all others should be FooTool, but heh).